lib/hal/video.c

```diff
@@ -198,6 +198,10 @@ BOOLEAN XVideoListModes(VIDEO_MODE *vm, int bpp, int refresh, void **p)
 	
 	DWORD dwAdapter = dwEnc & 0x000000FF;
 	DWORD dwStandard = dwEnc & 0x0000FF00;
+
+	ULONG eepromVidSize, eepromValueType, eepromReturnLength;
+
+	ExQueryNonVolatileSetting(0x08, &eepromValueType, (void*)&eepromVidSize, 4, &eepromReturnLength);
```

Comment:
This call is not necessary, `VIDEO_ENC_GET_SETTINGS` already retrieves this field. See https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/blob/a6c0353c5beaf14766313050bfc0c9fe222147ac/src/core/kernel/exports/EmuKrnlAv.cpp#L133

Reply:
Fixed, thanks!